react/src/components/FolderCreateModal.tsx

```diff
@@ -76,11 +78,23 @@ const FolderCreateModal: React.FC<FolderCreateModalProps> = ({
   const { message } = App.useApp();
 
   const formRef = useRef<FormInstance>(null);
+  const baiClient = useSuspendedBackendaiClient();
+  const userRole = useCurrentUserRole();
   const currentDomain = useCurrentDomainValue();
   const currentProject = useCurrentProjectValue();
 
   const baiRequestWithPromise = useBaiSignedRequestWithPromise();
 
+  const { data: allowedTypes, isLoading: isLoadingVhostInfo } =
+    useSuspenseTanQuery({
+      queryKey: ['allowedTypes', modalProps.open],
```

Comment:
`isFetching` and `isLoading` are different. Additionally, in `useTanSuspendedQuery`, both `isFetching` and `isLoading` are always false because the query is suspended during fetching/loading.

Reply:
Also I got an error.
```
["allowedTypes",false] data is undefined
    at Object.onSuccess (http://127.0.0.1:9081/static/js/bundle.js:210909:19)
    at resolve (http://127.0.0.1:9081/static/js/bundle.js:212068:25)
```